.github/workflows/python_requirements.yaml

```
@@ -0,0 +1,26 @@
+name: Python Requirements Dependency Check
+
+on:
+  push:
+    branches: ["main"]
```

```suggestion
    branches: ["main","master"]
```
Nur für mein Verständnis, das würde aktuell nicht triggern, wenn wir auf dem Master branch arbeiten?

---

Korrekt. Habs angepasst. Danke. Da "main" seit einiger Zeit der default auf github ist, hab ichs nicht gegen gecheckt.